### ISSUE TITLE
Add pipeline to download snapshots

### ISSUE THREAD

**Issue by jamesmbaazam** [jamesmbaazam/mpox_2024_nowcasting#3]
@sbfnk notes that the key thing for now will be to set up an automated download for snapshots (assuming past ones aren't available)

**Comment by seabbs** [comment]
I agree that is indeed the key thing!

**Comment by seabbs** [comment]
or finding out if anyone already has snapshots

**Comment by jamesmbaazam** [comment]
Notes:
- R script to download Africa CDC data from shiny dashboard: https://gist.github.com/spoonerf/5a76e743d603c6e8e15f003b14165226.
- Our world in data data sources: [owid/monkeypox](https://github.com/owid/monkeypox), especially the details in the README

**Comment by jamesmbaazam** [comment]
And this https://github.com/owid/monkeypox, especially the details in the README

**Comment by adamkucharski** [comment]
The global.health direct data link (from [website](https://reports.global.health/mpox-2024/)) is also here: https://mpox-2024.s3.eu-central-1.amazonaws.com/latest.csv – but not sure about update frequency.

**Comment by jamesmbaazam** [comment]
@seabbs do you know of similar work for other data pipelines (guessing some exist for covid-19).

**Comment by seabbs** [comment]
I know of internal CFA ones (i.e. not public) and the related ones from Global.health but not otherwise

**Comment by jamesmbaazam** [comment]
Would it suffice to set up a CRON actions that downloads the data at regular intervals and saves it in a directory?

**Comment by seabbs** [comment]
yes but for Global.health ones this is literally all they are already doing so maybe could use that in some places?

**Comment by jamesmbaazam** [comment]
I only see the snapshots from 2022/23 here: https://7rydd2v2ra.execute-api.eu-central-1.amazonaws.com/web/archive. Not sure they are doing same for the current outbreak.

**Comment by adamkucharski** [comment]
On a related note, I wondered if there's a package that can handle this e.g. point to a link or folder and automatically version and standardise snapshots? A bit like [orderly2](https://github.com/mrc-ide/orderly2) but for data streams.
